python/rmmbr/main_test.py

```diff
@@ -7,6 +7,14 @@
 from rmmbr import cloud_cache, local_cache, mem_cache
 
 
+async def wait_all_tasks():
```

Comment:
in theory you can also expose a `wait_writes` method from the lib

Reply:
trickier and i need a specific use case for the details